Library/Formula/openvpn.rb

```diff
@@ -42,11 +42,19 @@ def install
     rm "#{share}/doc/openvpn/README.polarssl"
   end
 
-  def caveats; <<-EOS.undent
-    If you have installed the Tuntap dependency as a source package you will
-    need to follow the instructions found in `brew info tuntap`. If you have
-    installed the binary Tuntap package, no further action is necessary.
+  def caveats
+    s = ""
```

Comment:
I think you need an `s` at the end of this block as well, for it to actually print. Maybe not. Your usage here is as echoed by Misty in the previous thread so I'll leave the review to Misty.